package.json

```
@@ -53,56 +52,54 @@
 	"devDependencies": {
 		"@phenomnomnominal/tsquery": "^6.1.3",
 		"@release-it/conventional-changelog": "^8.0.1",
-		"@types/eslint": "^8.56.0",
-		"@typescript-eslint/eslint-plugin": "^6.15.0",
-		"@typescript-eslint/parser": "6.20.0",
+		"@types/eslint": "^8.56.5",
+		"@typescript-eslint/eslint-plugin": "^7.1.1",
+		"@typescript-eslint/parser": "7.1.1",
 		"@typescript/vfs": "^1.5.0",
-		"@vitest/coverage-v8": "^1.1.0",
+		"@vitest/coverage-v8": "^1.3.1",
 		"console-fail-test": "^0.2.3",
-		"cspell": "^8.2.3",
-		"eslint": "^8.56.0",
+		"cspell": "^8.6.0",
+		"eslint": "^8.57.0",
 		"eslint-plugin-deprecation": "^2.0.0",
 		"eslint-plugin-eslint-comments": "^3.2.0",
-		"eslint-plugin-jsdoc": "^48.0.0",
-		"eslint-plugin-jsonc": "^2.11.2",
-		"eslint-plugin-markdown": "^3.0.1",
-		"eslint-plugin-n": "^16.5.0",
-		"eslint-plugin-no-only-tests": "^3.1.0",
-		"eslint-plugin-perfectionist": "^2.5.0",
-		"eslint-plugin-regexp": "^2.1.2",
-		"eslint-plugin-vitest": "^0.3.18",
-		"eslint-plugin-yml": "^1.11.0",
-		"husky": "^9.0.0",
+		"eslint-plugin-jsdoc": "^48.2.1",
+		"eslint-plugin-jsonc": "^2.13.0",
+		"eslint-plugin-markdown": "^4.0.1",
+		"eslint-plugin-n": "^16.6.2",
+		"eslint-plugin-package-json": "^0.12.0",
+		"eslint-plugin-perfectionist": "^2.6.0",
+		"eslint-plugin-regexp": "^2.3.0",
+		"eslint-plugin-vitest": "^0.3.24",
+		"eslint-plugin-yml": "^1.12.2",
+		"husky": "^9.0.11",
 		"jsonc-eslint-parser": "^2.4.0",
-		"knip": "^4.0.0",
-		"lint-staged": "^15.2.0",
+		"knip": "^5.0.3",
+		"lint-staged": "^15.2.2",
 		"markdownlint": "^0.33.0",
 		"markdownlint-cli": "^0.39.0",
-		"npm-package-json-lint": "^7.1.0",
-		"npm-package-json-lint-config-default": "^6.0.0",
 		"prettier": "^3.1.1",
-		"prettier-plugin-curly": "^0.1.3",
+		"prettier-plugin-curly": "^0.2.1",
 		"prettier-plugin-packagejson": "^2.4.7",
 		"release-it": "^17.0.1",
 		"sentences-per-line": "^0.2.1",
 		"should-semantic-release": "^0.2.1",
-		"tsup": "^8.0.1",
+		"tsup": "^8.0.2",
 		"typedoc": "^0.24.8",
 		"typedoc-plugin-coverage": "^2.2.0",
 		"typedoc-plugin-custom-validation": "^1.1.1",
 		"typedoc-plugin-konamimojisplosion": "^0.0.2",
 		"typedoc-plugin-mdn-links": "^3.1.8",
 		"typedoc-plugin-versions": "^0.2.4",
 		"typescript": "^5.3.3",
-		"vitest": "^1.1.0",
+		"vitest": "^1.3.1",
 		"yaml-eslint-parser": "^1.2.2"
 	},
 	"peerDependencies": {
 		"typescript": ">=4.2.0"
 	},
 	"packageManager": "pnpm@8.15.1",
 	"engines": {
-		"node": ">=16"
+		"node": ">=18"
```

This would be a breaking change and I don't want to do it just yet.

```suggestion
		"node": ">=16"
```